src/core/parser.zsh

```diff
@@ -168,3 +168,37 @@ _fzf_complete_parse_option_arguments() {
 
     echo - $result
 }
+
+_fzf_complete_parse_option() {
```

Comment:
Put this function before `_fzf_complete_parse_option_arguments` because it should be easier to find if the related functions are sorted from the shorter to the longer name.